js/index.js

```diff
@@ -120,6 +120,15 @@ function renderStats() {
   });
 }
 
+function renderEmoji(code) {
+  var template = document.getElementById('emoji-template').innerHTML;
+  return Mustache.render(template, {code: code});
+}
+
+function filterEmoji(str) {
+  return str.replace(/\:([a-z_+\-0-9]*)\:/g, renderEmoji('$1'));
```

Comment:
There are still issues with this, e.g.
https://regex101.com/r/bR4eY2/1

You need to exclude matches within HTML tags (e.g. positive lookahead and non-capturing group).